src/constants/enums/SystemIntakeContractStatus.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-shadow
```

Comment:
purely out of curiosity - what is this shadowing? (or what does `ts` think it's shadowing)

Reply:
So I was never sure why it does this, but eslint throws a no-shadow error whenever you try to use enums. I looked it up and found that apparently `no-shadow` just doesn't understand typescript enums. We can replace it with `@typescript-eslint/no-shadow` to make it work properly with typescript ([see issue](https://github.com/typescript-eslint/typescript-eslint/issues/2733)).

I'm going to put up a PR to make this change in the eslint config once this PR goes through. Thanks for bringing this up!

Reply:
oh wow that's really interesting! thanks for the info!